### PR TITLE
[MyRocksDD] Disable DESC for mysql.password_history when creating sys…

### PR DESCRIPTION
**Pull request by sunshine-Chun**
…tem table

Summary: Myrocks doesn't support DESC. Disable DESC when creating rocksdb system tables.

Test Plan: Don't break current MTR

Reviewers:

Subscribers:

Tasks:

Tags:

### CODE REVIEW

File: scripts/mysql_system_tables.sql [hunk 1]
```diff
@@ -26,6 +26,7 @@
 
 set @have_innodb= (select count(engine) from information_schema.engines where engine='INNODB' and support != 'NO');
 set @is_mysql_encrypted = (select ENCRYPTION from information_schema.INNODB_TABLESPACES where NAME='mysql');
+set @ddse= (select @@default_dd_storage_engine);
```
Comment (laurynas-biveinis) [inline scripts/mysql_system_tables.sql:29]: ddse == ROCKSDB && is_mysql_encrypted == true combination should be diagnosed and rejected?

Comment (sunshine-Chun) [reply scripts/mysql_system_tables.sql:29]: would you mind elaborating this? I didn't quite follow.

Comment (laurynas-biveinis) [reply scripts/mysql_system_tables.sql:29]: This same script allows creating an instance with mysql tablespace in InnoDB being encrypted, which MyRocks would not support. I believe we should return an error if the DD engine is MyRocks and mysql tablespace encryption is requested.

**Comment by sunshine-Chun** [comment]
closed.

**Pull request closed without merging**